### ISSUE TITLE
bitflyer private API URI is wrong

### ISSUE THREAD

**Issue by t234y678** [kroitor/ccxt#121]
bitflyer private API URI is wrong. Currently, ccxt calls https://api.bitflyer.jp/**me**/**v1**/xxx but it should call https://api.bitflyer.jp/**v1**/**me**/xxx

Here is the reference to bitflyer private API:
https://lightning.bitflyer.jp/docs?lang=en#http-private-api

Thanks!

```
---------------------------------------------------------------------------
HTTPError                                 Traceback (most recent call last)
/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in fetch(self, url, method, headers, body)
    282             opener = _urllib.build_opener(handler)
--> 283             response = opener.open(request, timeout=int(self.timeout / 1000))
    284             text = response.read()

/Users/xx/anaconda/lib/python3.6/urllib/request.py in open(self, fullurl, data, timeout)
    531             meth = getattr(processor, meth_name)
--> 532             response = meth(req, response)
    533 

/Users/xx/anaconda/lib/python3.6/urllib/request.py in http_response(self, request, response)
    641             response = self.parent.error(
--> 642                 'http', request, response, code, msg, hdrs)
    643 

/Users/xx/anaconda/lib/python3.6/urllib/request.py in error(self, proto, *args)
    569             args = (dict, 'default', 'http_error_default') + orig_args
--> 570             return self._call_chain(*args)
    571 

/Users/xx/anaconda/lib/python3.6/urllib/request.py in _call_chain(self, chain, kind, meth_name, *args)
    503             func = getattr(handler, meth_name)
--> 504             result = func(*args)
    505             if result is not None:

/Users/xx/anaconda/lib/python3.6/urllib/request.py in http_error_default(self, req, fp, code, msg, hdrs)
    649     def http_error_default(self, req, fp, code, msg, hdrs):
--> 650         raise HTTPError(req.full_url, code, msg, hdrs, fp)
    651 

HTTPError: HTTP Error 404: Not Found

During handling of the above exception, another exception occurred:

ExchangeNotAvailable                      Traceback (most recent call last)
<ipython-input-34-23c6021795c9> in <module>()
      1 cc = create_order('cc')
----> 2 bf = create_order('bf')

<ipython-input-33-8bbf3f018da8> in create_order(exchange)
     38         order = zaif.create_limit_buy_order('BTC/JPY',  0.005, 465000)
     39     elif exchange == 'bf':
---> 40         order = bf.create_limit_buy_order('BTC/JPY',  0.005, 465000)
     41 
     42     return order

/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in create_limit_buy_order(self, market, amount, price, params)
    685 
    686     def create_limit_buy_order(self, market, amount, price, params={}):
--> 687         return self.create_order(market, 'limit', 'buy', amount, price, params)
    688 
    689     def create_limit_sell_order(self, market, amount, price, params={}):

/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in create_order(self, market, type, side, amount, price, params)
   2393             'size': amount,
   2394         }
-> 2395         result = self.privatePostSendchildorder(self.extend(order, params))
   2396         return {
   2397             'info': result,

/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in request(self, path, api, method, params, headers, body)
   2423                 'Content-Type': 'application/json',
   2424             }
-> 2425         return self.fetch(url, method, headers, body)
   2426 
   2427 #------------------------------------------------------------------------------

/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in fetch(self, url, method, headers, body)
    316             elif e.code in [401, 422, 511]:
    317                 error = AuthenticationError
--> 318             self.raise_error(error, url, method, e, details)
    319         except _urllib.URLError as e:
    320             self.raise_error(ExchangeNotAvailable, url, method, e)

/Users/xx/anaconda/lib/python3.6/site-packages/ccxt/__init__.py in raise_error(self, exception_type, url, method, error, details)
    253                 method,
    254                 url,
--> 255                 details,
    256             ]))
    257         else:

ExchangeNotAvailable: bitflyer POST https://api.bitflyer.jp/me/v1/sendchildorder 404 Not Found  WnF}'ذ@)("F.b\+/w%QKԇA^7FJj˪;ggfoGyx(1@3T]y8yqH28~FS5J%M8*MlH؈وѹ19aCv%tnX+BLZ-	z%)+!=CFdLl9Fc
H0xI6p@z!+),Y_0B_	>gѰŔوwaER库ɤő8ui*~:4((kA%qB %$#6VP$?RehyUs*;|0]V*ey&NSFȡ~M$["<s{'1Qg{	
#ޕ$Qj%NERZ9Pֲp<B~\`B0,0=[FTDLևIbZ vI0E>ꖹ^3xZ>_g^tA*RJa]  _QS3(KI)O8i}ʨ"Q^NZH(yud:}*tk3-EGk%(REC]J],$Lna5v~{	O|[؅gYjƘ.%fYIK3!^Qr8UW%u{ˣ0[k7#4v;x]!t\S1PS Wg5I]JӟI㪕V,ۿFуb#	l*/CK$;¿|nTW卧'eLpǯ(#j_^6
```

**Comment by t234y678** [comment]
This should work.
```
    def request(self, path, api='public', method='GET', params={}, headers=None, body=None):
        request = '/' + self.version + '/' + path
        if api == 'private':
            # request = '/me' + request
            request = '/' + self.version + '/me/' + path
        url = self.urls['api'] + request

```

**Comment by kroitor** [comment]
Will fix this very soon! Thx!

**Comment by kroitor** [comment]
Fixed in 1.4.15+, thanks again!
